beers/models.py

```diff
@@ -80,3 +80,99 @@ class Manufacturer(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class Beer(models.Model):
+    name = models.CharField(max_length=25, db_index=True)
+    style = models.ForeignKey(
+        BeerStyle, models.DO_NOTHING, related_name='beers',
+        # TODO: prevent this being null?
```

Comment:
It would seem reasonable to prevent it from being NULL, but it would probably be more upfront work when a new Beer is discovered and can't quickly be sorted into a style (maybe a moderation queue?)

Reply:
I like the moderation queue idea. Since Untappd definitely does not use BJCP styles, we'll probably have to map those by hand. If/when they add new ones, we'll have to map those by hand.

Reply:
#30 filed to track that